File: discover_ruby/write_them_to_disc.rb
require 'httpclient'

def req(url, query, extheaders)
	client = HTTPClient.new
	client.get_content(url, query, extheaders)
end

extheaders = { 'User-Agent' => 'Holberton_School', 'Authorization' => 'token fa51959c07bf48159fbaa3c6800b12efb49f217e' }
File.open('/tmp/40', 'w') do |file|
	file.write( req('https://api.github.com/search/repositories?q=language:ruby&sort=stars&order=desc', nil, extheaders) )
end
puts "The file was saved!"

File: discover_more_python/write_them_to_disc.py
import urllib2

def http_get(url, headers):
	req = urllib2.Request(url, None, headers)
	response = urllib2.urlopen(req)
	return response.read()
	
request_headers = { 'User-Agent': 'Holberton_School', 'Authorization': 'token fa51959c07bf48159fbaa3c6800b12efb49f217e' }
file_object = open('/tmp/40', 'w')
file_object.write(http_get('https://api.github.com/search/repositories?q=language:python&sort=stars&order=desc', request_headers))
file_object.close()
print "The file was saved!"
